cloud_logger/cloudlogger/cloudlog_handler.py

```diff
@@ -0,0 +1,30 @@
+from logging import Handler, LogRecord
+import os
+import platform
+
+from cloudlog_commons import LogSender, Log, LogType
+
+class CloudLogHandler(Handler):
+    sender: LogSender
+    app_name: str
+
+    def __init__(self, app_name: str, logging_endpoint=None):
+        Handler.__init__(self=self)
+
+        self.app_name = app_name
+        self.sender = LogSender(logging_endpoint)
```

Comment:
Log sender should not be a part of log handler. It should be instantiated outside of this class.

Reply:
What do you mean? Do you want me to move it above Handler class implementation and then access it directly, or should I convert LogSender class to singleton/weak singleton(it will dispose when not uset) so every handler instance has access to the same Sender instance?

Reply:
handler should push the logs on the queue. It should not know about the sender all it needs to know is the queue.